pages/linux/homectl.md

```diff
@@ -0,0 +1,36 @@
+# homectl
+
+> Create, remove, change or inspect home directories using the systemd-homed service.
+> More information: <https://man.archlinux.org/man/homectl.1>.
+
+- List user accounts and their associated home directories:
+
+`homectl list`
+
+- Create a user account and the associated home directory:
+
+`sudo homectl create {{username}}`
```

Comment:
This is rather misleading in  that `systemd-homed` also manages creating user accounts, rather than just managing home directories. Despite this, there's little we can do about that so we don't need to change this example at all - just wanted to voice my concerns about systemd-homed in general.

Is there provision for creating system accounts with home directories here? And how does this interact with the ability for users to login or not (e.g. via SSH)?

Reply:
Oh you write so much :joy_cat:

Well, if a user is not logged in and it has `.ssh/authorized_keys`, when you try to login remotely using SSH the `systemd-homed` request for the user password because (encrypted or not) the services coulnd't read the `authorized_keys` file.
There is a workaround to associate a `authorized_keys` file outside the user's home folder here: https://wiki.archlinux.org/index.php/Systemd-homed#SSH_remote_unlocking but you still need the user password to login the account itself :upside_down_face:. And when you logout from the SSH session, the account goes to `not login` states and the password is needed again :tired_face: .